frontends/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:20.12-alpine
```

Comment:
It might be nice to use something with bash. I think people bash into containers somewhat regularly. `sh` works, too, though.

Reply:
I normally just use `sh` to poke around containers, but let me know if you'd like me to install `bash`.